app/src/main/java/com/x8bit/bitwarden/ui/platform/components/snackbar/BitwardenSnackbarHost.kt

```diff
@@ -1,35 +1,32 @@
 package com.x8bit.bitwarden.ui.platform.components.snackbar
 
-import androidx.compose.material3.Snackbar
 import androidx.compose.material3.SnackbarHost
-import androidx.compose.material3.SnackbarHostState
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
 
 /**
  * A custom Bitwarden-themed snackbar.
  *
- * @param hostState The state of this snackbar.
- * @param modifier The [Modifier] to be applied to this radio button.
+ * @param bitwardenHostState The state of this snackbar.
+ * @param modifier The [Modifier] to be applied to the [SnackbarHost].
  */
 @Composable
 fun BitwardenSnackbarHost(
-    hostState: SnackbarHostState,
+    bitwardenHostState: BitwardenSnackbarHostState,
     modifier: Modifier = Modifier,
 ) {
     SnackbarHost(
-        hostState = hostState,
+        hostState = bitwardenHostState.snackbarHostState,
         modifier = modifier,
-    ) {
-        Snackbar(
-            snackbarData = it,
-            shape = BitwardenTheme.shapes.snackbar,
-            containerColor = BitwardenTheme.colorScheme.background.alert,
-            contentColor = BitwardenTheme.colorScheme.text.reversed,
-            actionColor = BitwardenTheme.colorScheme.background.alert,
-            actionContentColor = BitwardenTheme.colorScheme.icon.reversed,
-            dismissActionContentColor = BitwardenTheme.colorScheme.icon.reversed,
-        )
+    ) { snackbarData ->
+        val message = snackbarData.visuals.message
+        val currentCustomSnackbarData = bitwardenHostState.currentSnackbarData
+        if (currentCustomSnackbarData?.key == message) {
+            BitwardenSnackbar(
+                bitwardenSnackbarData = currentCustomSnackbarData,
+                onDismiss = { snackbarData.dismiss() },
+                onActionClick = { snackbarData.performAction() },
```

Comment:
Do we need these outer lambdas, can it just be `onDismiss = snackbarData.dismiss`

Reply:
changing to `snackbarData::dismiss` should be fine, they aren't value fields themselves